R/ui-legacy.R

```diff
@@ -1,9 +1,22 @@
-#' User interface
+#' Legacy functions related to user interface
```

Comment:
All of the existing exported `ui_*()` functions are now marked as superseded and there is no remaining use of anything in this file in usethis. As far as usethis is concerned, I could delete this. But I can't because other people are using the `ui_*()` functions.